### PR TITLE
Explore Component 

### PR DESCRIPTION
**Pull request by martypowell**
- Adds Explore Link
- Adds Explore List (displays a list of explore links)

based on xd.

image

### CODE REVIEW

**Comment by martypowell** [comment]
this needs to be exported.

**Comment by martypowell** [comment]
> this needs to be exported.

this has been exported.

File: src/styles/partials/components/_explore.scss [hunk 1]
```diff
@@ -0,0 +1,56 @@
+.dg_explore-link {
+  background: $gray-lightest;
+  display: block;
+  font-family: $default-heading-font;
+  font-weight: 600;
+  font-size: $smallest-heading-font-size;
+  min-height: 130px;
+  margin-bottom: $default-margin;
```
Comment (tmccoy529) [inline src/styles/partials/components/_explore.scss:8]: do we have variables for font weight and height?

File: src/styles/partials/components/_site-header.scss [hunk 2]
```diff
@@ -39,20 +39,34 @@
...
 @media screen and (max-width: $header-breakpoint) {
+  #search-form {
+    display: none;
+    padding: 15px;
+    position: absolute;
+    top: 85px;
+    height: 65px;
+    width: 100%;
+    background: white;
+    z-index: 999999999;
```
Comment (tmccoy529) [inline src/styles/partials/components/_site-header.scss]: same as above. Should try to use variables for values here if possible

File: src/styles/partials/components/_site-header.scss [hunk 3]
```diff
@@ -70,6 +84,13 @@
...
+
+  #search-form {
+    .dg_search-btn {
+      right: 30px;
+      top: 20px;
+    }
+  }
 }
 
 $small-mobile-button-size: 65px;
```
Comment (tmccoy529) [inline src/styles/partials/components/_site-header.scss]: your search form uses 65px as well below. Could this be named something more generic to accommodate that as well?

 #search-form {
    top: 65px;
  }

Comment (martypowell) [reply src/styles/partials/components/_site-header.scss]: good catch, yea, i think its just header-height. Will update.

File: src/styles/partials/components/_explore.scss (hunk 1 above)
Comment (martypowell) [reply src/styles/partials/components/_explore.scss:8]: yes, will update.

Comment (martypowell) [reply src/styles/partials/components/_explore.scss:8]: height, this is a one off, i will add a comment.

File: src/styles/partials/components/_explore.scss [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  justify-content: flex-start;
+  flex-wrap: wrap;
+
+  .dg_explore-link {
+    margin-right: $default-margin;
+    width: 30%;
+  }
+}
+
+@media screen and (max-width: $medium-breakpoint) {
```
Comment (theilman7) [inline src/styles/partials/components/_explore.scss:42]: Between 769px and 897px, the items appear to stack two across. May consider adding a larger breakpoint.

File: src/styles/partials/components/_site-header.scss [hunk 5]
```diff
@@ -39,20 +39,34 @@
...
-  .dg_search-container {
-    display: none;
+  #search-form.is-visible {
+    display: flex;
   }
 
   .dg_mobile-search-button {
     display: block;
     padding: $default-padding;
+    z-index: $max-z-index-minus-1;
```
Comment (theilman7) [inline src/styles/partials/components/_site-header.scss]: Is the `z-index` property necessary? The button does not appear to change in an adverse way when I remove it.

Comment (martypowell) [reply src/styles/partials/components/_site-header.scss]: @theilman7 if its not there, you cannot select the search icon on smaller screens. At least taht is what happened for me, can you test your change as see if thats consistent for you?

File: src/styles/partials/components/_explore.scss (hunk 4 above)
Comment (martypowell) [reply src/styles/partials/components/_explore.scss:42]: will take a look.

File: src/styles/partials/components/_explore.scss (hunk 1 above)
Comment (martypowell) [reply src/styles/partials/components/_explore.scss:8]: added comment, resolving.

File: src/styles/partials/components/_site-header.scss (hunk 2 above)
Comment (martypowell) [reply src/styles/partials/components/_site-header.scss]: jsut relaized these shouldn't be showing up in this pr.

File: src/styles/partials/components/_site-header.scss (hunk 3 above)
Comment (martypowell) [reply src/styles/partials/components/_site-header.scss]: jsut relaized these shouldn't be showing up in this pr.

File: src/styles/partials/components/_site-header.scss (hunk 5 above)
Comment (martypowell) [reply src/styles/partials/components/_site-header.scss]: these shouldn't be in this pr

File: src/styles/partials/components/_explore.scss (hunk 4 above)
Comment (martypowell) [reply src/styles/partials/components/_explore.scss:42]: @theilman7 per our dicussion, this is looking ok i think, please just double check on yoru machine.

Comment (theilman7) [reply src/styles/partials/components/_explore.scss:42]: Closing issue. Will log a bug if the issue shows up when we view on template. @martypowell

**Review by theilman7 — APPROVED** [review]
Looks good.

**Pull request merged**